internal/server/cel.go

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
Most of this file can be removed in favor of https://github.com/fluxcd/pkg/pull/858